bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -99,15 +99,26 @@ spec:
         livenessProbe:
           exec:
             command:
-            - /usr/share/openvswitch/scripts/ovs-ctl
-            - status
+            - /bin/bash
+            - -c
+            - |
+              #!/bin/bash
+              /usr/share/openvswitch/scripts/ovs-ctl status > /dev/null &&
+              /usr/bin/ovs-appctl ofproto/list > /dev/null &&
+              /usr/bin/ovs-vsctl show > /dev/null &&
+              if ovs-vsctl br-exists br0; then ovs-ofctl -O OpenFlow13 probe br0; else true; fi
```

Comment:
ovs-ofctl(1) says: "`With the -t or --timeout option, [probe] can test whether an OpenFlow switch or controller is up and running.`" which kind of suggests we should probably use that? (Maybe in the `ovs-vsctl` and `ovs-appctl` commands too?)

Reply:
Yes - best to use the '-t ' option because if some error happens during the command execution and ovs-vswitchd or the controller doesn't respond, the command will hang.

Reply:
very good suggestion. added timeouts.